packages/tbds-utilities/lib/inline-icon.scss

```diff
@@ -0,0 +1,25 @@
+$_icon-margin: .2em;
+$_visually-center-offset: -.1em;
+
+.tbds-inline-icon {
+  height: 1em;
+  position: relative;
+  top: $_visually-center-offset;
+  vertical-align: middle;
```

Comment:
Instead of `relative` positioning, with `top` and `vertical-align`, thoughts on using flexbox?

Reply:
Flexbox would solve the alignment issue, but that it would require another class to the container element. The idea I'm running with here is that the container shouldn't need any knowledge about the icon it contains. This makes the markup simpler, and there's no refactoring that has to be done if the icon is removed: simply remove the element with this class, no stray classes on the parent.